pygotham/frontend/talks.py

```diff
@@ -94,10 +98,24 @@ def proposal(pk=None):
         )
 
     form = TalkSubmissionForm(obj=talk)
+
     if form.validate_on_submit():
         form.populate_obj(talk)
 
+        speaker = Speaker(
+            user=current_user,
+            talk=talk,
+            primary=True,
+            confirmed_at=datetime.now(),
```

Comment:
Please use `arrow.utcnow().to('America/New_York').naive` to be consistent with other parts of the codebase here.